### DISCUSSION TITLE
Valorant doesn't work with AtlasOS plz help me

### DISCUSSION THREAD

**Discussion by NisMaaa** [Atlas-OS/Atlas#1301] [Help & Support]
When i start Valorant, the game closes itself after 5 minutes. I've done everything i can to fix the problem but nothing worked. So i decided to contact the valorant support, they gave me a bunch of solutions to fix the problem, but nothing worked. And then the support sent me one last message; Valorant doesn't work with AtlasOS. 
The thing is, i know i'm not the only one with AtlasOS that want to play valorant but everytime i search if someone have the same problem as me i get nothing, no results at all. So if someone has had this problem and managed to fix it, PLZ HELP ME.

**Comment by Assationater** [comment]
You dont get a secure boot or tpm error message from val? 
Did you follow the atlas install to a tee?

**Reply by Zxab3340** [reply]
https://github.com/orgs/Atlas-OS/discussions/1723#discussion-10770556 help😔

**Comment by Ast3risk-ops** [comment]
You should have an error message somewhere. I would recommend staying away from the game due to the fact that the anticheat is a known spyware that sends desktop screenshots and system information and mouse movements and keyboard keys to RIOT (and their parent Tencent) all the time even if the game isn't running. It's also a backdoor for RIOT and attackers to access your PC.

However I assume you love the game too much to care, so I'd recommend double checking the requirements manually. Oh, and don't get the riot client from Epic Games.

**Reply by Zxab3340** [reply]
https://github.com/orgs/Atlas-OS/discussions/1723#discussion-10770556 help😔

**Comment by s0nic1** [comment]
Just turn on Secure Boot, Trusted Computing, and Core Isolation. That's your fix. I been playing valorant with AtlasOS with no issues

**Reply by malalim1** [reply]
I did all of those and I still have issues with Valorant, it just gets stuck launching

**Reply by s0nic1** [reply]
> I did all of those and I still have issues with Valorant, it just gets stuck launching

whats ur pc specs? and have u tried to ran valorant with 1.1.1.1?

**Reply by malalim1** [reply]
> > I did all of those and I still have issues with Valorant, it just gets stuck launching
> 
> whats ur pc specs? and have u tried to ran valorant with 1.1.1.1?

RTX 2050, i5-11400H and 8gigs of ram. and yeah i tried it with cloudflare

**Reply by s0nic1** [reply]
> > > I did all of those and I still have issues with Valorant, it just gets stuck launching
> > 
> > 
> > whats ur pc specs? and have u tried to ran valorant with 1.1.1.1?
> 
> RTX 2050, i5-11400H and 8gigs of ram. and yeah i tried it with cloudflare

Are you on Windows 11?

**Reply by malalim1** [reply]
> > > > I did all of those and I still have issues with Valorant, it just gets stuck launching
> > > 
> > > 
> > > whats ur pc specs? and have u tried to ran valorant with 1.1.1.1?
> > 
> > 
> > RTX 2050, i5-11400H and 8gigs of ram. and yeah i tried it with cloudflare
> 
> Are you on Windows 11?

the windows 11 version of atlas yeah

**Reply by itz-rj-here** [reply]
Idk, but I feel that you have an important corrupted file somewhere, which is interfering with Valorant software.

**Reply by Lodesthard** [reply]
I cant activate secure boot how do you do that without troubleshot> Just turn on Secure Boot, Trusted Computing, and Core Isolation. That's your fix. I been playing valorant with AtlasOS with no issues

**Reply by Zxab3340** [reply]
https://github.com/orgs/Atlas-OS/discussions/1723#discussion-10770556 help😔

**Comment by Sivatech24** [comment]
The Vanguard anti-cheat system requires certain official Windows security features to be enabled. These features may be removed or modified when using the Atlas OS playbook or a custom Windows ISO, which can cause Vanguard to stop working.

**Reply by Zxab3340** [reply]
https://github.com/orgs/Atlas-OS/discussions/1723#discussion-10770556 help😔

**Comment by Pipeee-txt** [comment]
Hello, I had the same issue, and I fixed it by changing a few settings in the BIOS. Everything in my BIOS was set to default, so I enabled UEFI and Secure Boot. In my BIOS, it went like this:For UEFI: Go to Windows OS Configuration or Boot, and change CSM/Legacy to UEFI.For Secure Boot: Set Secure Boot Support to Enabled. If it says "Not Active," go to Key Management, change it to Custom, select Enroll all Factory Default Keys, and then change it back to Standard.The second thing I needed to do was enable TPM 2.0. In the BIOS, I went to Security $\rightarrow$ Trusted Computing, and set Security Device Support to Enabled. Under Device Selection, I chose PTT in my case because I have an Intel CPU.After saving the changes, you can verify it in Windows:Press Win + R, type tpm.msc, and press Enter. If TPM shows as "Ready for use," you're good to go.If not, open PowerShell and run Get-Tpm. If TpmReady returns True, you're all set.That worked for me!

**Reply by Zxab3340** [reply]
https://github.com/orgs/Atlas-OS/discussions/1723#discussion-10770556 help😔

**Comment by Zxab3340** [comment]
https://github.com/orgs/Atlas-OS/discussions/1723#discussion-10770556 help😔
